### PR TITLE
RavenDB-16264 - fix DeleteBlobs from Azure

### PR DESCRIPTION
**Pull request by grisha-kotler**
https://issues.hibernatingrhinos.com/issue/RavenDB-16264

### CODE REVIEW

File: src/Raven.Server/Documents/PeriodicBackup/Azure/RavenAzureClient.cs [hunk 1]
```diff
@@ -585,22 +585,46 @@ IEnumerable<BlobProperties> GetResult()
...
+                    continue;
+
+                if (response.StatusCode == HttpStatusCode.NotFound)
+                    continue;
+
+                throw StorageException.FromResponseMessage(response);
             }
+        }
+
+        private void DeleteMultipleBlobs(List<string> blobs)
```
Comment (ayende) [inline src/Raven.Server/Documents/PeriodicBackup/Azure/RavenAzureClient.cs:607]: Who is calling this now?

Comment (grisha-kotler) [reply src/Raven.Server/Documents/PeriodicBackup/Azure/RavenAzureClient.cs:607]: This is disabled for now

**Comment by ppekrol** [comment]
- What about 4.2?
- Please switch all Azure tests to use real credentials, not emulator
- Please make sure that set of tests for Azure matches the AWS and GCP ones

**Comment by ppekrol** [comment]
What is the status of this? I see that some of the tests are commented out.

File: test/StressTests/Server/Documents/PeriodicBackup/Retention.cs [hunk 2]
```diff
@@ -1,4 +1,5 @@
-﻿using System.Threading.Tasks;
```
Comment (ppekrol) [inline test/StressTests/Server/Documents/PeriodicBackup/Retention.cs:1]: Why this is commented out?

Comment (garayx) [reply test/StressTests/Server/Documents/PeriodicBackup/Retention.cs:1]: didnt notice it, I have to add stress tests for azure as well

**Comment by grisha-kotler** [comment]
LGTM

**Pull request merged**